frontend/src/metabase/query_builder/components/NativeQueryEditor/utils.ts

```diff
@@ -12,6 +12,7 @@ function getLinesForHeight(height: number) {
 }
 
 const FRACTION_OF_TOTAL_VIEW_HEIGHT = 0.4;
+const FULL_HEIGHT = 500;
```

Comment:
Unfortunately, the query editor has to be a fixed pixel height and we can't just set it to flex or 100%. We could revisit this, but it would require introducing a conditional around the resizeable component, which is pretty deeply integrated into the query editor.